sdf/forward_camera.sdf

```diff
@@ -92,8 +78,8 @@
         <plugin name="MotionPlugin" filename="libMotionPlugin.so">
           <link name="base_link">
             <period>10</period>
-            <linear_vel>0.2, 0, 0</linear_vel>
```

Comment:
I ran into this problem. It took me a while to realize that the syntax was incorrect.